.github/workflows/dotnet.yml

```diff
@@ -58,6 +69,7 @@ jobs:
       run: nuget pack .build/MvvmCross.Plugin.BLE.nuspec -Version $(git describe)
     - name: Upload packages
       uses: actions/upload-artifact@v2
+      if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
```

Comment:
So this only uploads the artifacts for pushes to master? Any particular reason why you apply that restriction? nupkgs from a PR can be quite useful for people wanting to try a branch with an external project that references Plugin.BLE. I've done this myself a number of times.

Reply:
Let me remove i didn't know what was the right way. I use devops for everythign so my system is very different.